### ISSUE TITLE
Please bump major version and release

### ISSUE THREAD

**Issue by isaacs** [npm/npm-registry-mock#25]
b5c9b0f is a breaking change, and it'd be nice to pull in the fixes to npm, but it does require shuffling how the args are handled.

Another option is to back out b5c9b0f, and release without a major version bump.  Either way is fine.

**Comment by robertkowalski** [comment]
will take care... 

btw not sure if you know about it - i think you should be able to publish - feel free to add more owner

**Comment by robertkowalski** [comment]
just noticed: i blatantly lied. but now you are an owner.

**Comment by isaacs** [comment]
@robertkowalski It's ok, I still have server admin powers, so not to worry ;)

**Comment by robertkowalski** [comment]
@isaacs it is getting late here and I would need a review on https://github.com/npm/npm-registry-mock/pull/26 (the feature was a bad idea but we were expecting cases emulating a 404 automatically would make sense) to proceed further. i also fixed the builds for 0.8 in another PR.

I would also really like to upgrade to the latest hock, which would make testing with `https` possible:

```javascript
    var http = require('http'),
        hock = require('hock'),
        request = require('request');

    var mock = hock.createHock();
    mock
        .get('/some/url')
        .reply(200, 'Hello!');

    var server = http.createServer(mock.handler);
    server.listen(1337, function () {
        request('http://localhost:' + 1337 + '/some/url', function(err, res, body) {
           console.log(body);
        });
    });

```
